addons/sale/report/sale_report.py

```diff
@@ -66,10 +66,10 @@ def _get_done_states(self):
     qty_delivered = fields.Float(string="Qty Delivered", readonly=True)
     qty_to_invoice = fields.Float(string="Qty To Invoice", readonly=True)
     qty_invoiced = fields.Float(string="Qty Invoiced", readonly=True)
-    price_subtotal = fields.Monetary(string="Untaxed Total", readonly=True)
-    price_total = fields.Monetary(string="Total", readonly=True)
-    untaxed_amount_to_invoice = fields.Monetary(string="Untaxed Amount To Invoice", readonly=True)
-    untaxed_amount_invoiced = fields.Monetary(string="Untaxed Amount Invoiced", readonly=True)
+    price_subtotal = fields.Float(string="Untaxed Total", readonly=True)
+    price_total = fields.Float(string="Total", readonly=True)
+    untaxed_amount_to_invoice = fields.Float(string="Untaxed Amount To Invoice", readonly=True)
+    untaxed_amount_invoiced = fields.Float(string="Untaxed Amount Invoiced", readonly=True)
```

Comment:
What's exactly the problem with those fields being monetary fields?
They are holding a monetary amount in the currency of the 'record', which is basically the meaning of a monetary field :thinking: